src/pages/steps/CalendarStep.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTranslation } from 'react-i18next';
+
+import { FormElement } from '@/ui/FormElement';
+import { Inline } from '@/ui/Inline';
+import { StackProps } from '@/ui/Stack';
+import { Typeahead } from '@/ui/Typeahead';
+
+import { FormDataUnion, StepsConfiguration } from './Steps';
+
+type CalendarStepProps = StackProps;
+
+const CalendarStep = ({ ...props }: CalendarStepProps) => {
+  const { t, i18n } = useTranslation();
+
+  const getHourOptions = () => {
+    const hours = Array(24).fill(0);
+    const quarterHours = ['00', '15', '30', '45'];
+    const times = [];
+    hours.forEach((_hour, i) => {
+      quarterHours.forEach((quarterHour) =>
+        times.push(`${i > 9 ? i : `0${i}`}:${quarterHour}`),
+      );
+    });
+    return times;
+  };
+
+  const hourOptions = getHourOptions();
```

Comment:
```suggestion
  const hourOptions = useMemo(getHourOptions, []);
  ```

Reply:
Why would you use "useMemo"? I think it's not necessary here? 
I can just move this const outside the component function?

Reply:
Yes that's true 😄